chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -365,7 +365,10 @@ integration-citationDialog-details-done =
     .label = { general-done }
 integration-citationDialog-details-showInLibrary = Show in Library
 integration-citationDialog-settings-title = Citation Settings
-integration-citationDialog-lib-no-matches = No selected, opened, or cited items match the current query.
+integration-citationDialog-lib-no-items = { $search ->
+    [true] No selected, opened, or cited items match the current search
+   *[other] No selected, or opened items 
```

Comment:
→ "No selected, opened, or cited items"

Reply:
But if the query is empty, there can not be any cited items though? Or does it sound better that way?

Reply:
Oh, because we don't show cited items when there's no search. OK, then it should be "No selected or open items" (no comma, "opened" → "open" in both messages).

Reply:
Ok, cool! "No selected or open items" and "No selected, open, or cited items match the current search". Sometimes grammar is quite hard...